roles/unbind-keycloak-apb/tasks/main.yml

```diff
@@ -20,13 +18,14 @@
     var: keycloak_auth_response
     verbosity: 2
 
--
-  name: delete client {{ _apb_bind_creds.resource }} in realm {{ namespace }}
+- set_fact: KEYCLOAK_REALM={{ _apb_bind_creds.bearer_installation.realm}}
+
+- name: delete client {{ _apb_bind_creds.resource }} in realm {{ KEYCLOAK_REALM }}
   uri:
-    url: "{{ keycloak_protocol }}://{{ keycloak_route.stdout }}/auth/admin/realms/{{ namespace }}/clients/{{ _apb_bind_creds.resource }}"
+    url: "{{ keycloak_route.stdout }}/auth/admin/realms/{{ KEYCLOAK_REALM }}/clients/{{ _apb_bind_creds.client_id }}"
```

Comment:
Here you use `_apb_bind_creds.client_id`. Is it same as `_apb_bind_creds.resource`?